CMSgov/onemac
# Oy2 17916

Story: https://qmacbis.atlassian.net/browse/OY2-17916
Endpoint: https://do5yi0gi5vg6e.cloudfront.net

### Details

Add additional hint text to medicaid spa id entry field.

### Changes

- made hint text a string array in the TransmittalNumber component and to display each hint on a new line
- converted each instance of idHintText to string array

### Test Plan

1. Login as state submitter
2. From the Submission Dashboard select the "New Submission" button and choose to create a medicaid SPA
3. Verify new hint text appears within the "SPA ID" field
